### DISCUSSION TITLE
Docker / Sail / general environment question

### DISCUSSION THREAD

**Discussion by sidewaysglance** [magic-test/magic-test-laravel#35] [Q&A]
Hi all,

What's recommended in terms of development environment to enable this magic? I am on a Apple M1 device, I am guessing it will be a PITA to configure on there ...

Do you have any suggestions for environment? Anything that will not work? Other gotchas? Am happy to stand up a vm on a separate dev server / ubuntu with gui installed etc, ideally using sail or laradock, or ... homestead? I haven't used that before. 

Whatever is easiest to get up and running, last resort is apt installing all separate requirements from scratch, although willing to do so if it makes getting substantial e2e test writing done.

Thank you

**Comment by sidewaysglance** [comment]
Found this, nevermind: https://github.com/magic-test/magic-test-laravel/discussions/14#discussioncomment-481375

**Reply by mateusjatenee** [reply]
thanks -- i'll add it to the docs 👍

**Reply by sidewaysglance** [reply]
To be honest I’m still struggling with it. If good docs existed for docker integration it would be very useful.

Alternatively another (possibly opinionated) recommended setup to get things moving would be fine too.
